modules/nwtc-library/src/NWTC_Num.f90

```diff
@@ -55,6 +55,13 @@ MODULE NWTC_Num
    REAL(ReKi)                                :: TwoByPi                       !< 2/Pi
    REAL(ReKi)                                :: TwoPi                         !< 2*Pi
 
+   REAL(SiKi)                                :: Pi_R4                         !< Ratio of a circle's circumference to its diameter in 4-byte precision
```

Comment:
Justa  suggestion, I believe pi can be defined as a parameter like this:
```
real(SiKi), parameter :: Pi_R4 = acos(-1.0_SiKi) 
```

Reply:
Yes, except that unfortunately doesn't work for all compilers.